demo/website/src/App.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import ProjectsList from './components/ProjectsList';
+import elasticsearch from 'elasticsearch';
 
 import './App.css';
 
 const App = () => (
   <div className="App">
     <h1>EUBFR Website</h1>
-    <ProjectsList />
+    <ProjectsList elasticsearch={elasticsearch} />
```

Comment:
Is there a specific reason to pass an npm module as a prop? I think it's ok to include it in the `ProjectsList.js` as no obvious need to listen for rendering events on this type of object?